pandas/tests/arrays/interval/test_interval.py

```diff
@@ -103,3 +105,110 @@ def test_repr():
         "Length: 2, closed: right, dtype: interval[int64]"
     )
     assert result == expected
+
+
+# ----------------------------------------------------------------------------
+# Arrow interaction
+
+
+pyarrow_skip = td.skip_if_no("pyarrow", min_version="0.15.1.dev")
```

Comment:
is this right?

Reply:
As mentioned above, the pandas -> arrow part already works on 0.15 in general, but due to a "bug" in pyarrow (due to the use of `.values`, see https://github.com/apache/arrow/pull/5753), period and interval ExtensionArrays specifically don't work yet (because `.values` returns an object array for those, and not an EA). So therefore those tests for period and interval also only work with pyarrow higher than 0.15.